apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/Operations.java

```diff
@@ -235,7 +235,8 @@ public void runRetention(
   }
 
   private Path getTrashPath(String path, String filePath, String trashDir) {
-    return new Path(filePath.replace(path, new Path(path, trashDir).toString()));
+    return new Path(
+        filePath.replace(new Path(path).toString(), new Path(path, trashDir).toString()));
```

Comment:
why did we need this change?

Reply:
With the schema, the string `path` was `file:///tmp/...`. However, we were trying to replace it with something like `file:/tmp/...`, because the `Path` type only allows for single forward slashes. So, I converted it to a `Path` to standardize the format to `file:/tmp/...` so they would match.